src/cli/reset.js

```diff
@@ -118,6 +119,10 @@ async function resetThemeTo(themeId) {
 
 async function resetPlugin(pluginId) {
 	try {
+		if (!nconf.get('plugins:active')) {
+			winston.error('Cannot reset plugins while plugin state is set in the configuration (config.json, environmental variables or terminal arguments), please modify the configuration instead');
+			process.exit(1);
+		}
```

Comment:
Should this not be a negation? Only exit if plugins are defined in configuration...